cfme/tests/control/test_rest_control.py

```diff
@@ -14,6 +14,7 @@
 )
 from cfme.utils.version import current_version
 from cfme.utils.wait import wait_for
+from manageiq_client.api import APIException
```

Comment:
Please move this above the internal imports
from manageiq_client.api import APIException

from cfme.utils.version import current_version
from cfme.utils.wait import wait_for